vector/src/main/java/im/vector/app/features/home/room/detail/search/SearchResultController.kt

```diff
@@ -54,10 +54,6 @@ class SearchResultController @Inject constructor(
         fun loadMore()
     }
 
-    init {
-        setData(null)
-    }
```

Comment:
I count 9 other controllers with the same code, do you think the same fix should be applied?

<img width="577" alt="image" src="https://user-images.githubusercontent.com/3940906/144240470-8647a0a8-28ea-4773-a2dd-8311fe1bb5bb.png">

Reply:
I think yes, they should be removed. I will update the PR and test those screens